ui/src/app/api/auth/[...nextauth]/route.ts

```diff
@@ -0,0 +1,37 @@
+import NextAuth from "next-auth/next";
+import GitHubProvider from "next-auth/providers/github"
+import CredentialsProvider from "next-auth/providers/credentials"
+
+export const authOptions = {
+    providers: [
+        GitHubProvider({
+            clientId: process.env.GITHUB_ID ?? "",
+            clientSecret: process.env.GITHUB_SECRET ?? ""
+        }),
+        CredentialsProvider({
+            name: "Credentials",
+            credentials: {
+                email: {label: "Email", type: "text", placeholder: "email"},
+                password: {label: "Password", type: "password"}
+            },
+            async authorize(credentials, req) {
+                const res = await fetch('http://localhost:3333/auth/login', {
+                    method: "POST",
+                    body: JSON.stringify(credentials),
+                    headers: {"Content-Type": "application/json"}
+                });
+                const user = await res.json();
+
+                if (res.ok && user) {
+                    console.log(user.access_token)
```

Comment:
console logs in code are bad practice, there should be eslint rule added for that: https://eslint.org/docs/latest/rules/no-console